content/doc/book/installing/_docker.adoc

```diff
@@ -301,31 +247,16 @@ Your `<docker-container-name>` can be obtained using the `docker ps` command.
 
 == Accessing the Jenkins home directory
 
-There is a possibility you may need to access the Jenkins home directory, for
-instance, to check the details of a Jenkins build in the `workspace`
-subdirectory.
+There is a possibility you may need to access the Jenkins home directory, for instance, to check the details of a Jenkins build in the `workspace` subdirectory.
 
-If you mapped the Jenkins home directory (`/var/jenkins_home`) to one on your
-machine's local file system (i.e. in the `docker run ...` command
-<<downloading-and-running-jenkins-in-docker,above>>), then you can access the
-contents of this directory through your machine's usual terminal/command prompt.
+If you mapped the Jenkins home directory (`/var/jenkins_home`) to one on your machine's local file system (i.e. in the `docker run ...` command <<downloading-and-running-jenkins-in-docker,above>>), then you can access the contents of this directory through your machine's usual terminal/command prompt.
 
-Otherwise, if you specified the `--volume jenkins-data:/var/jenkins_home` option in
-the `docker run ...` command, you can access the contents of the Jenkins home
-directory through your container's terminal/command prompt using the
-link:https://docs.docker.com/engine/reference/commandline/container_exec/[`docker container exec`]
-command:
+Otherwise, if you specified the `--volume jenkins-data:/var/jenkins_home` option in the `docker run ...` command, you can access the contents of the Jenkins home directory through your container's terminal/command prompt using the link:https://docs.docker.com/engine/reference/commandline/container_exec/[`docker container exec`] command:
 
 `docker container exec -it <docker-container-name> bash`
 
-As mentioned <<accessing-the-jenkins-console-log-through-docker-logs,above>>,
-your `<docker-container-name>` can be obtained using the
-link:https://docs.docker.com/engine/reference/commandline/container_ls/[`docker container ls`]
-command. If you specified the
-`--name jenkins-blueocean` option in the `docker container run ...` 
-command above (see also
-<<accessing-the-jenkins-blue-ocean-docker-container,Accessing the Jenkins/Blue
-Ocean Docker container>>), you can simply use the `docker container exec` command:
+As mentioned <<accessing-the-jenkins-console-log-through-docker-logs,above>>, your `<docker-container-name>` can be obtained using the link:https://docs.docker.com/engine/reference/commandline/container_ls/[`docker container ls`] command.
+If you specified the `--name jenkins-blueocean` option in the `docker container run ...`  command above (see also <<accessing-the-jenkins-blue-ocean-docker-container,Accessing the Jenkins/Blue Ocean Docker container>>), you can simply use the `docker container exec` command:
```

Comment:
You may want to check these links to make sure they work.

Reply:
It does appear that the links go to the correct sections/external sites from what I've tested. Was there one that you had noticed was not working/leading back to the correct anchor?